docs/contribute/building.md

```diff
@@ -67,15 +55,14 @@ resulting website show up on a locally-served web server.
     }
     ```
```

Comment:
Why is this text highlighted in red?

Reply:
I think it's angry about the json; I'll fix it.